src/test/java/com/baidu/hugegraph/api/TraverserApiTest.java

```diff
@@ -913,6 +915,71 @@ public void testCustomizedCrosspoints() {
         Assert.assertTrue(expectedVids.containsAll(vertices));
     }
 
+    @Test
+    public void testNeighborRank() {
+        initNeighborRankGraph();
+
+        RankRequest.Builder builder = new RankRequest.Builder();
+        builder.source("O");
+        builder.steps().direction(Direction.OUT).degree(-1).top(10);
+        builder.steps().direction(Direction.OUT).degree(-1).top(10);
+        builder.steps().direction(Direction.OUT).degree(-1).top(10);
+        builder.alpha(0.9).capacity(-1);
+        RankRequest request = builder.build();
+
+        List<Map<Object, Double>> ranks = neighborRankAPI.post(request);
+        Assert.assertEquals(4, ranks.size());
+        Assert.assertEquals(ImmutableMap.of("O", 1.0D), ranks.get(0));
+
+        Assert.assertEquals(ImmutableMap.of("B", 0.45075000000000004D,
+                                            "A", 0.3D,
+                                            "C", 0.3D),
+                            ranks.get(1));
+        Assert.assertEquals(ImmutableMap.builder()
+                                        .put("G", 0.17550000000000002D)
+                                        .put("H", 0.17550000000000002D)
+                                        .put("E", 0.135D)
+                                        .put("F", 0.135D)
+                                        .put("I", 0.135D)
+                                        .put("J", 0.135D)
+                                        .build(),
+                            ranks.get(2));
+        Assert.assertEquals(ImmutableMap.of("M", 0.15795D,
+                                            "K", 0.12150000000000001D,
+                                            "L", 0.12150000000000001D),
+                            ranks.get(3));
```

Comment:
remove added vertices and edges, otherwise affecting scan vertex/edge tests results.